observation_portal/requestgroups/serializers.py

```diff
@@ -228,6 +228,8 @@ class Meta:
     def validate_instrument_configs(self, value):
         if [instrument_config.get('fill_window', False) for instrument_config in value].count(True) > 1:
             raise serializers.ValidationError(_('Only one instrument_config can have `fill_window` set'))
+        if len(set([instrument_config.get('rotator_mode', '') for instrument_config in value])) > 1:
+            raise serializers.ValidationError(_('Rotator modes within the same configuration must be the same'))
```

Comment:
is this a limitation of the site software implementation?

Reply:
Yeah, site software will use rotator mode in the first inst config and ignore the rest of them. So, technically, we could allow a user to pass in different rotator modes, but I think that would just end up being confusing if they don't get what they thought they asked for...

Reply:
From Mark: "We do not support multiple rotator modes or angles. The mode and angle in the first Instrument Configuration will be used for all associated Instrument Configurations. This is because we do not want to introduce an axis slew between configurations which would blow-up our overheads estimate".

Reply:
interesting...